conf/lookup/BCD-Port.csv

```diff
@@ -14910,7 +14908,7 @@ PortId,CountryCode,PortType,PortCode,KeyWords,Name,Name-cy,Priority,Region,Locat
 46888,UK,3,NDY,,Sanday Island Cruise Port Seaport - NDY,Sanday Island Cruise Port Seaport - NDY,,1,NULL,0,
 46889,UK,3,OBN,,Oban Seaport - OBN,Oban Seaport - OBN,,1,NULL,0,
 46890,UK,3,PEN,,Penzance Seaport - PEN,Penzance Seaport - PEN,,1,NULL,0,
-46891,UK,3,PLY,,The Port of Plymouth - PLY,The Port of Plymouth - PLY,,1,NULL,0,
+46891,UK,3,PLY,Padstow,The Port of Plymouth - PLY,The Port of Plymouth - PLY,,1,NULL,0,

```

Comment:
Delete 32755,UK,3,PAD,,Padstow Seaport - PAD,Padstow Seaport - PAD,,1,NULL,1,


Now part of this

Reply:
Just keep both